app/controllers/experiments/list.js

```diff
@@ -107,9 +106,19 @@ export default Ember.Controller.extend({
                     sequence: []
                 }
             });
-            this.send('toggleModal');
-            newExperiment.save().then(function() {
-                self.transitionToRoute('experiments.info', newExperiment.id);
+            newExperiment.save();
+            newExperiment.on('didCreate', () => {
```

Comment:
You're defining a callback in a callback and then setting that callback and calling that callback?